test/unit/spec/EnrollQuestions_spec.js

```diff
@@ -228,7 +227,9 @@ function (Q, Okta, OktaAuth, Util, Form, Beacon, Expect, Router, RouterUtil, Bro
         spyOn(RouterUtil, 'isDocumentVisible').and.callFake(function () {
           return true;
         });
-        return tick();
+        // document.removeEventListener happens to be last call in the flow
+        // before invoke global.successFn.
+        return Expect.waitForSpyCall(document.removeEventListener);
```

Comment:
Would it work to wait for the global.successFn?

Reply:
ah, good point!